jekyll/__glossary/project.md

```diff
@@ -0,0 +1,5 @@
+---
+term: Project
+---
+
+A CircleCI project shares the name of the code repository for which it automates workflows, tests, and deployment. It is visible on the Projects page of the CircleCI app and must be added with the Add Project button. After the project is added, it is possible to configure settings, contexts, environment variables, and team members who may follow the project. Following a project enables a user to subscribe to email notifications for the project build status and adds the project to their CircleCI dashboard.
```

Comment:
While the information about adding and following a project is certainly useful, doesn't this go beyond the scope of merely defining a term?

Reply:
It does, but I think it is helpful and we only really talka bout it in one other place, so I'm going to leave it for now.